scipy/optimize/tests/test_linprog.py

```diff
@@ -687,6 +687,75 @@ def cb(xk, **kwargs):
         assert_(callback_complete[0])
         assert_allclose(last_xk[0], res.x)
 
+    def test_bug_8662(self):
```

Comment:
Rather than adding this test here, consider putting it in the `LinprogCommonTests` class. That way it will be run against `method='interior-point'`, too.